test/runtests.jl

```diff
@@ -85,4 +85,9 @@ end
 
 end
 
+@testset "unwrap" begin
+    @test DataAPI.unwrap(1) === 1
+    @test DataAPI.unwrap(missing) === missing
```

Comment:
Ah, clever: with that fallback we don't need to use `passmissing(unwrap)`!

Reply:
Yes!